src/main/kotlin/nl/avisi/structurizr/site/generatr/site/SiteGenerator.kt

```diff
@@ -7,14 +7,13 @@ import nl.avisi.structurizr.site.generatr.includedSoftwareSystems
 import nl.avisi.structurizr.site.generatr.site.model.*
 import nl.avisi.structurizr.site.generatr.site.views.*
 import java.io.File
+import java.math.BigInteger
 import java.nio.file.Path
+import java.security.MessageDigest
 
 fun copySiteWideAssets(exportDir: File) {
```

Comment:
Could you change the order of this function and its companion `copySiteWideAsset`, so the `copySiteWideAsset` function is directly below the `copySiteWideAssets` function?

See https://marekhudyma.com/code-style/2021/03/02/step-down-rule.html

Reply:
Done, I think I had briefly public before private in my mind ;)